Bio/SwissProt/__init__.py

```diff
@@ -62,6 +62,25 @@ class Record(object):
         - seqinfo           tuple of (length, molecular weight, CRC32 value)
         - sequence          The sequence.
 
+        Example:
```

Comment:
That should be about four spaces further to the left I think, likewise the doctest which follows. Double check in your favourite reStructuredText editor - or an online one?